contents/handbook/strategy/strategy.md

```diff
@@ -4,40 +4,43 @@ sidebar: Handbook
+  * We can go for breadth over depth in the product, similar to GitLab, letting the community fill in the gaps.
+  * Being open core means we can aim for mass adoption with our free version, without worrying about competition from companies offering hosting services.
 * Tighter integration with developers' environments.
- * Integration with git repo (feature flagging) & integration of data into local environment (terminal/localhost)
- * Working with existing databases and pipelines.
- * 1 click / 1 line install into production.
-* Defensive product-team strategy - stay at parity with their features, but don't innovate here.
- * This means execution is much faster.
- * It will prevent most churn due to missing features from major tools that product teams will push for.
+  * Integration with git repo (feature flagging) & integration of data into local environment (terminal/localhost)
+  * Working with existing databases and pipelines.
+  * 1-click / 1-line install into production.
+* Defensive Product team strategy - stay at parity with their features, but don't innovate here.
+  * This means execution is much faster.
+  * It will prevent most churn due to missing features from major tools that product teams will push for.
 
 ## How will we be the best?
 
-* Engineering-led culture. Engineers want to deal with engineers.
-* We will make the company remote-first, to encourage the greatest possible degree of community collaboration.
+* **Engineering-led culture:** Engineers want to deal with engineers.
+* **Remote first:** We will make the company remote-first, to encourage the greatest possible degree of community collaboration.
 
 ## What are our capabilities, and what else is needed to succeed on this path?
 
 * We currently have a strong product for hobbyists or startups. We have enterprise interest but the adoption is weak.
-* We need greater scalability for larger, greater value, enterprise customers to go into production. This will be key for either an adoption or revenue based Series A.
+* We need greater scalability for larger, greater value, enterprise customers to go into production. This will be key for either an adoption or revenue-based Series A.
 * We need a more innovative, developer-specific side to the product for all user types to move forward. This is what will make us truly best in class, and is also a pre-requisite to earn the right to impress teams in enterprises who demand the latest and greatest.
 
 ## What are the systems and metrics that empower our capabilities?
 
-* Adoption: MAUs is something we should focus on. A tiny bit of product work needed to make that happen. Underneath this, we should start to look at the % of deployed:in production - that is our greatest area of drop off.
+* Adoption: MAUs is something we should focus on. A tiny bit of product work needed to make that happen. Underneath this, we should look at the percentage of deployed instances in production - that is our greatest area of drop off.
```

Comment:
We now have MAUs being tracked. So we could delete from 'A tiny bit'... through to the end of this paragraph